include/ideep/operators/conv.hpp

```diff
@@ -766,16 +776,15 @@ struct convolution_forward
                       prop_kind aprop_kind = prop_kind::forward,
                       const lowp_kind alowp_kind = u8s8,
                       const engine& aengine = engine::cpu_engine()) {
+    // Consider fp32 only for IPEX
     if (bias.is_empty()) {
       compute_dispatch</*with_bias=*/false, true, true>(
           src, weights, bias, dst_dims, dst, strides, dilates,
-          padding_l, padding_r, groups, src_scales, weights_scales, dst_scales,
-          IDEEP_EMPTY_ZP, IDEEP_EMPTY_ZP, attr, aalgorithm, aprop_kind, alowp_kind, aengine);
+          padding_l, padding_r, groups, attr, aalgorithm, aprop_kind, alowp_kind, aengine);
     } else {
       compute_dispatch</*with_bias=*/true, true, true>(
           src, weights, bias, dst_dims, dst, strides, dilates,
-          padding_l, padding_r, groups, src_scales, weights_scales, dst_scales,
-          IDEEP_EMPTY_ZP, IDEEP_EMPTY_ZP, attr, aalgorithm, aprop_kind, alowp_kind, aengine);
+          padding_l, padding_r, groups, attr, aalgorithm, aprop_kind, alowp_kind, aengine);
```

Comment:
Should we remove the scales arguments from the function prototype too?

Reply:
These are deprecated APIs that are still used in IPEX. For example:
https://github.com/intel-innersource/frameworks.ai.pytorch.ipex-cpu/blob/c558a512406a588bb664c3f98bf5427ce806796d/intel_extension_for_pytorch/csrc/aten/cpu/Conv.cpp#L67
Thus, the scales arguments cannot be removed.

Reply:
Then, why can't we fix the caller too?

Reply:
If we remove the scales arguments, it becomes the new API, which is already available after the refactoring.
I think that might be the next step to use new APIs on the framework side. For now, we keep compatibility for both frameworks (PT & IPEX). Next would be removing deprecated APIs from caller. Third, removing deprecated APIs from ideep.
Do you think it is OK?

Reply:
Sounds good. Could you please record it as a task in github or JIRA?

Reply:
Sure. We may use [this JIRA](https://jira.devtools.intel.com/browse/IPB-996) to track it.